pyexasol/connection.py

```diff
@@ -203,16 +285,54 @@ def __init__(self
     def execute(self, query, query_params=None) -> ExaStatement:
         """
         Execute SQL query with optional query formatting parameters
-        Return ExaStatement object
+
+        Args:
+            query:
+                SQL query text, possibly with placeholders
+            query_params:
+                 Values for placeholders
+
+        Returns:
+            ExaStatement object
+
+        Examples:
+
+            >>> con = ExaConnection(...)
+            >>> con.execute(
+            ...        query="SELECT * FROM {table!i} WHERE col1={col1}",
+            ...        query_params={'table': 'users', 'col1':'bar'}
+            ...)
         """
         return self.cls_statement(self, query, query_params)
 
     def execute_udf_output(self, query, query_params=None):
         """
         Execute SQL query with UDF script, capture output
-        Return ExaStatement object and list of Path-objects for script output log files
 
-        Exasol should be able to open connection to the machine where current script is running
+        Note:
+            Exasol should be able to open connection to the machine where current script is running.
```

Comment:
```suggestion
            Exasol should be able to open connection to the machine where the output listener socket is running.
```

Reply:
Or maybe:
```suggestion
            Exasol should be able to open connection to the machine where `pyexasol` is executed.
```

Reply:
From what I understand this would change the actual meaning it is not about where pyexasol is executed, rather the script (on DB).